index.js

```diff
@@ -29,59 +28,67 @@ module.exports = {
           return aposModule.__meta.name.includes(nativeModule);
         });
 
-        if (isCustomModule && aposModule.schema && aposModule.schema.length) {
-          const moduleTypeInA2 = aposModule.__meta.chain
-            .reverse()
-            .find(element => Object.keys(moduleTypes).find(type => element.name === type));
+        if (!isCustomModule || aposModule?.schema.length) {
```

Comment:
Node 14 is still a supported release and this is meant to run inside an existing A2 project, so we shouldn't be relying on Node 16-only functionality like `?.` here.

Reply:
It's supposed to be available in Node 14 https://node.green/#ES2020-features-optional-chaining-operator-----

Reply:
You're right, carry on. It's Node 12 I usually worry about because a lot of A2 sites are on Node 12 servers still, but for migration purposes the developer can reasonably be asked to run Node 14 locally.